### PR TITLE
Bicep Quickstart Sample to provision Azure Redis for Cache as Vector DB

### PR DESCRIPTION
**Pull request by manjitsin**
# PR Checklist

Check these items before submitting a PR... 

[Contribution Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md)

[Best Practice Guide](https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md)

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

## Changelog

* This is a quickstart sample to provision the Azure Cache for Redis service.
* The Redis Cache contains the `Redis Search` module which allows the Redis cache to be used as a Vector DB.
* Once provisioned, user can use the redis to store embeddings, create indexes and perform semantic and hybrid search.

### CODE REVIEW

File: quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep [hunk 1]
```diff
@@ -0,0 +1,44 @@
+@description('Location of all resources')
+param location string = resourceGroup().location
+
+@description('Name of the Redis Enterprise Cache')
+param redisCacheName string = ''
```
Comment (alex-frankel) [inline quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep:5]: Can't have a default value that will fail the deployment

File: quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+@description('SKU of the Redis Enterprise Cache')
+param redisCacheSKU string = 'Enterprise_E10'
+
+@description('Capacity of the Redis Enterprise Cache')
+param redisCacheCapacity int
```
Comment (alex-frankel) [inline quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep:11]: need a parameters file to provide a value for this. That is part of why validation is failing.

**Review by alex-frankel — CHANGES_REQUESTED** [review]

File: quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep (hunk 1 above)
Comment (manjitsin) [reply quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep]: I have updated the value. Please review.

File: quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep (hunk 2 above)
Comment (manjitsin) [reply quickstarts/microsoft.cache/redis-enterprise-vectordb/main.bicep]: Updated. Added the parameters file.

**Pull request merged**